README.md

```diff
@@ -1,40 +1,44 @@
 <div align="center">
 
-  <h1><code>near-bindgen</code></h1>
+  <h1><code>near-sdk</code></h1>
 
   <p>
     <strong>Rust library for writing NEAR smart contracts.</strong>
   </p>
+  <p>
+    Previously known as <code>near-bindgen</code>.
+  </p>
+
 
   <p>
-    <a href="https://crates.io/crates/near-bindgen"><img src="https://img.shields.io/crates/v/near-bindgen.svg?style=flat-square" alt="Crates.io version" /></a>
-    <a href="https://crates.io/crates/near-bindgen"><img src="https://img.shields.io/crates/d/near-bindgen.svg?style=flat-square" alt="Download" /></a>
+    <a href="https://crates.io/crates/near-sdk"><img src="https://img.shields.io/crates/v/near-sdk.svg?style=flat-square" alt="Crates.io version" /></a>
+    <a href="https://crates.io/crates/near-sdk"><img src="https://img.shields.io/crates/d/near-sdk.svg?style=flat-square" alt="Download" /></a>
     <a href="https://discord.gg/gBtUFKR"><img src="https://img.shields.io/discord/490367152054992913.svg" alt="Join the community on Discord" /></a>
-    <a href="https://travis-ci.com/nearprotocol/near-bindgen"><img src="https://travis-ci.com/nearprotocol/near-bindgen.svg?branch=master" alt="Travis Build" /></a>
+    <a href="https://travis-ci.com/nearprotocol/near-sdk"><img src="https://travis-ci.com/nearprotocol/near-sdk.svg?branch=master" alt="Travis Build" /></a>
```

Comment:
This also needs to be fixed.

Reply:
@nearmax Is this Travis link is left pointing to nearprotocol on purpose?

Reply:
Nope, a mistake. I asked @ailisp to give me instruction on how to fix CI now that I have transferred it, I will fix it as a part of that.